### ISSUE TITLE
Feature request: ignore mode

### ISSUE THREAD

**Issue by infokiller** [philc/vimium#2297]
I'm looking for a way to toggle whether vimium is active **for the current session**.
My use case is that on some sites I want to have vimium active most of the time, but when I don't and want to use the native site keybindings insert mode is too cumbersome to use, since the first esc exits it.

**Comment by smblott-github** [comment]
You've seen [this](https://github.com/philc/vimium/wiki/Disabling-Vimium), yes?

Also, if you follow the tip right at the bottom of that page, then:

- `Ctrl-Shift-v`, `Ctrl-Enter` disables Vimium on the current site, and
- `Ctrl-Shift-v`, `Tab`, `Enter`, `Ctrl-Enter` re-enables it.

That sounds like a mouthful, but it's more natural than it sounds when written down.

**Comment by infokiller** [comment]
Yes, but that's indeed more keystrokes and less flexible than I'd hope for.
Ideally, I'd be able to toggle it just with `Alt-v` or something like that.
